common/deps.bzl

```diff
@@ -2,31 +2,18 @@ load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 
 def rules_python():
-    git_repository(
+    http_archive(
         name = "rules_python",
-        remote = "https://github.com/bazelbuild/rules_python.git",
-        tag = "0.1.0",
-        patches = [
-            # Force rules_python to export the requirements.bzl file in
-            # order for stardoc to be able to load it during documentation
-            # generation.
-            "@vaticle_bazel_distribution//:bazelbuild_rules_python-export-requirements-bzl-for-stardoc.patch",
-        ],
-        patch_args = ["-p1"],
+        sha256 = "84aec9e21cc56fbc7f1335035a71c850d1b9b5cc6ff497306f84cced9a769841",
+        strip_prefix = "rules_python-0.23.1",
+        url = "https://github.com/bazelbuild/rules_python/releases/download/0.23.1/rules_python-0.23.1.tar.gz",
     )
 
 def rules_pkg():
-    http_archive(
+    git_repository(
```

Comment:
NOTE: this _has_ to be a `git_repository`, so we can access the `bzl_library` rules to generate our own `stardoc` documentation. If we use `http_archive`, we don't see to have access to the `doc_build` package.